setup.cfg

```diff
@@ -0,0 +1,37 @@
+[metadata]
+name = django-pg-extensions
+version = 0.1.8
+author = Apostolos Bessas
+author_email = mpessas@gmail.com
+description = Extensions for Django to fully utilize PostgreSQL
+url = https://github.com/mpessas/django-pg-extensions
+license_file = LICENSE
+
+[options]
+include_package_data = True
+packages = find:
+python_requires = >=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*
+install_requires =
+    Django==1.8.18
+    six==1.16.0
+
+[options.packages.find]
+exclude =
+    tests*
+
+[tool:pytest]
+DJANGO_SETTINGS_MODULE = djangopg.test_settings
+addopts = -q
+
+[tox:tox]
+envlist = py27
+
+[testenv]
+install_command = pip install -q --no-index --find-links=file:///app/pip-cache {opts} {packages}
+commands = pytest -q --disable-warnings --cov-report term-missing --cov=djangopg tests
+deps =
+    pytest==4.6.11
+    pytest-cov==2.8.1
+    pytest-django==3.10.0
+    mock==1.0.1
```

Comment:
We're no longer mocking, we don't need to install this, do we?

Reply:
`mock` as well as `six` (as direct requirement)